ui/components/src/Table/model/table-model.ts

```diff
@@ -19,10 +19,12 @@ import {
   CoreOptions,
   RowData,
   RowSelectionState,
+  SortingState,
 } from '@tanstack/react-table';
 import { CSSProperties } from 'react';
 
 export type TableDensity = 'compact' | 'standard';
+export type SortDirection = 'asc' | 'desc' | undefined;
```

Comment:
i see SortDirection defined in tanstack as well, should we use that instead?

Reply:
If I recall correctly, there's  a sort direction defined in both the MUI Table and the TanStack table that are slightly different, so having an explicit definition here helps us have a single source of truth we control.